python/paddle/distribution.py

```diff
@@ -929,28 +931,28 @@ def log_prob(self, value):
         """Log probabilities of the given category. Refer to ``probs`` method.
 
         Args:
-          value (Tensor): The input tensor represents the selected category index.
+            value (Tensor): The input tensor represents the selected category index.
 
         Returns:
-          Tensor: Log probability.
+            Tensor: Log probability.
         
         Examples:
-        .. code-block:: python
+            .. code-block:: python
 
-          import paddle
-          from paddle.distribution import Categorical
+                import paddle
+                from paddle.distribution import Categorical
 
-          x = paddle.rand([6])
-          print(x.numpy())
-          # [0.32564053, 0.99334985, 0.99034804,
-          #  0.09053693, 0.30820143, 0.19095989]
+                x = paddle.rand([6])
+                print(x.numpy())
+                # [0.32564053, 0.99334985, 0.99034804,
```

Comment:
Better add `paddle.manual_seed(xx)` here, otherwise, users cannot get the same random output as your sample code.
Same for all the other examples.

Reply:
done